primer/react
# refactor(Banner): update region to use a dedicated aria-label

Closes https://github.com/github/primer/issues/3262



### Changelog



#### New



#### Changed



- Update Banner based on the changes to the API doc so that it is no longer `aria-labelledby` the heading and is instead `aria-label` with the variant info

#### Removed



### Rollout strategy




### Testing & Reviewing



- Verify that the change reflects the API docs changes
- Make sure that test scenarios have been accurately updated to reflect the changes
- Is there anything I'm missing in this implementation? 👀
